### ISSUE TITLE
Firebase emulator does not start inside a docker container when websocketPort is specified

### ISSUE THREAD

**Issue by fixl** [firebase/firebase-tools#5094]
### [REQUIRED] Environment info

**firebase-tools:** 11.14.1

**Platform:** Docker (alpine 3.1.6)

### [REQUIRED] Test case

With the following `Dockerfile`:

```Dockerfile
FROM alpine:3.16

RUN apk add --no-cache \
        openjdk11-jre-headless \
        nodejs \
        npm \
    && npm install -g firebase-tools
```

and `firebase.json`:

```json
{
    "emulators": {
        "ui": {
            "enabled": true,
            "host": "0.0.0.0",
            "port": 4000
        },
        "firestore": {
            "host": "0.0.0.0",
            "port": "8081",
            "websocketPort": "54321"
        }
    }
}
```

### [REQUIRED] Steps to reproduce

Build & run:

```bash
docker build . --tag firebase-test

docker run --rm -it -v $(pwd):/src -w /src firebase-test sh

# Inside the docker container
firebase --project test emulators:start
```

### [REQUIRED] Expected behavior

Firebase emulators should start up without issues. Exposing ports with `-p 54321:54321` in the above command doesn't change the behaviour either.

### [REQUIRED] Actual behavior

Emulators shut down with a **port taken** error:

```
⚠  firestore: Port 54321 is not open on 0.0.0.0, could not start websocket server for Firestore emulator.
⚠  firestore: To select a different port, specify that port in a firebase.json config file:
      {
        // ...
        "emulators": {
          "firestore": {
            "host": "HOST",
            ...
            "websocketPort": "WEBSOCKET_PORT"
          }
        }
      }
i  emulators: Shutting down emulators.

Error: Could not start websocket, port taken.

```

**Comment by yuchenshi** [comment]
I believe this should also be fixed in v11.14.3, but let us know otherwise

**Comment by fixl** [comment]
It works now. Cheers.
